APMrover2/Rover.h

```diff
@@ -42,16 +42,16 @@
 #include <AP_Terrain/AP_Terrain.h>
 #include <PID/PID.h>            // PID library
 #include <RC_Channel/RC_Channel.h>     // RC Channel Library
-#include <AP_RangeFinder/AP_RangeFinder.h>	// Range finder library
-#include <Filter/Filter.h>			// Filter library
-#include <Filter/Butter.h>			// Filter library - butterworth filter
+#include <AP_RangeFinder/AP_RangeFinder.h>  // Range finder library
+#include <Filter/Filter.h>            // Filter library
+#include <Filter/Butter.h>            // Filter library - butterworth filter
 #include <AP_Buffer/AP_Buffer.h>      // FIFO buffer library
-#include <Filter/ModeFilter.h>		// Mode Filter from Filter library
-#include <Filter/AverageFilter.h>	// Mode Filter from Filter library
+#include <Filter/ModeFilter.h>        // Mode Filter from Filter library
+#include <Filter/AverageFilter.h>    // Mode Filter from Filter library
 #include <AP_Relay/AP_Relay.h>       // APM relay
 #include <AP_ServoRelayEvents/AP_ServoRelayEvents.h>
-#include <AP_Mount/AP_Mount.h>		// Camera/Antenna mount
-#include <AP_Camera/AP_Camera.h>		// Camera triggering
+#include <AP_Mount/AP_Mount.h>        // Camera/Antenna mount
+#include <AP_Camera/AP_Camera.h>        // Camera triggering
```

Comment:
This comments have an inconsistent spacing. Since we are changing it, can you make them all have the same spacing (or be aligned)?